js/index.js

```diff
@@ -1,31 +1,49 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
+  const price = product.querySelector('.price span');
 
-  //... your code goes here
+  const quantityParent = product.querySelector('.quantity');
+  const quantityChildValue = quantityParent.firstElementChild.value;
+
+  const subtotal = product.querySelector('.subtotal span');
+
+  let priceSubtotal = price.innerText * quantityChildValue;
+  return (subtotal.innerText = priceSubtotal);
```

Comment:
you always do this, returning the assigment operation -> I personally find this weird, because in some other languages it doesn't work that way, because assignments don't return the right hand side and this would then return undefined.